src/Agent.Listener/Configuration/ConfigurationManager.cs

```diff
@@ -397,8 +397,16 @@ public async Task ConfigureAsync(CommandSettings command)
 
             agentSettings.NotificationSocketAddress = command.GetNotificationSocketAddress();
 
+            // Test to see if disableLogUpload and enabledLogOutput are both selected
+            if (command.GetDisableLogUploads() && command.GetReStreamLogsToFiles())
+            {
+                throw new NotSupportedException(StringUtil.Loc("You cannot use --disableloguploads and --reStreamLogsToFiles at the same time!"));
```

Comment:
Please place the line to en-US loc strings and use the line key here, as you made for: https://github.com/microsoft/azure-pipelines-agent/pull/4829/files#diff-9a438f6277d4fb97e75fb0ddbb30d16477d710b558b47a1a7d07d95f91ea9ff6R287
Instead, Loc function will not find the key and will show the broken message

Reply:
Just committed the change.